src/main/java/uk/gov/justice/probation/courtcaseservice/application/FeatureFlags.java

```diff
@@ -19,8 +19,8 @@ public FeatureFlags() {
         return flags;
     }
 
-    public boolean attendanceData() {
-        return flags.getOrDefault("fetch-attendance-data", true);
```

Comment:
Has fetch-attendance-data gone completely ? Was this flag supposed to just get rid of attendance data ?

Reply:
If so, could you get rid of all the occurrences of that flag ? It's still in some YAML and test code